module/VuFind/src/VuFind/Search/Base/Params.php

```diff
@@ -1064,23 +1064,24 @@ public function getFilterList($excludeCheckboxFilters = false)
     }
 
     /**
-     * Check for delimited facets -- if $field is a delimited facet field,
-     * process $displayText accordingly. Return the appropriate display value.
+     * Get a display text for a facet field.
      *
-     * @param string $field       The facet
-     * @param string $displayText The facet value
+     * @param string $field Facet field
+     * @param string $value Facet value
      *
      * @return string
      */
-    public function checkForDelimitedFacetDisplayText($field, $displayText)
+    public function getFacetValueDisplayText(string $field, string $value): string
```

Comment:
I've extracted this change to #2394 since I think it's a separate issue from the rest of the work here, and we should try to resolve in in the dev branch to avoid introducing an extra backward compatibility break between newly-added 8.1 functionality and subsequent 9.0 code. We should resolve that PR before merging this one, so I'll add a TODO checkbox at the top here.